src/core/internal/ParentedView.tsx

```diff
@@ -107,8 +107,25 @@ const ParentedView = forwardRef(function ParentedView(
       if (newContainer) {
         interactor.bindEvents(newContainer);
       }
+      return true;
+    }
+    return false;
```

Comment:
Agreed, it would be nice to have a setContainer method on the RenderWindowInteractor. It also follows the same pattern we have for the webgl/webgpu-specifc RenderWindows.